frontend/ui/screensaver.lua

```diff
@@ -16,7 +19,8 @@ local logger = require("logger")
 local _ = require("gettext")
 local Screen = Device.screen
 local T = require("ffi/util").template
-
+local TextBoxWidget = require("ui/widget/textboxwidget")
+local TopContainer = require("ui/widget/container/topcontainer")
```

Comment:
Last little thing: can you move these TextBoxWidget and TopContainer above between ScreenSaverWidget and UIManager, and keep an empty line after T.